docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "Bruno",
+  "type": "collection"
+}
\ No newline at end of file
```

Comment:
It's a good practice to add a newline at the end of the file. This can prevent potential issues with certain tools that might not handle files without a trailing newline correctly. [medium]